### ISSUE TITLE
Finding a private key from a file

### ISSUE THREAD

**Issue by Bitcoin228** [ishristov/bitcoin-private-key-fixer#19]
Good day, I ask you to help with launching a file to search for a private key from a file (s) without the last 4 characters.
Thanks in advance.....

**Comment by ishristov** [comment]
The file is there. You just have to download the repository with node and follow the instructions. It can recover up to 8-9 characters if they are at the end.

**Comment by Bitcoin228** [comment]
I have a key file that does not have the last 4 characters, can I run the whole file or insert only one key to check?

**Comment by ishristov** [comment]
Yes, the provided script in the repository will work for 4 missing chars at the end.

**Comment by iamchaay** [comment]
Hello, can we find up to 11 characters missing at the end?

**Comment by ishristov** [comment]
> Hello, can we find up to 11 characters missing at the end?

I doubt it - only if you have a super fast machine and are super super lucky. For 8 chars (at the end) it takes a few minutes on an old macbook. Let's say its 1 min on a super fast processor. But with each extra char it takes ~60 times longer. So for 11 chars it can take up to `60*60*60` => `1 hour*60*60` -> `a few months` non stop calculations.

**Comment by DAN-256** [comment]
can it work in IBM Quantum cloud 127 Qubit? How possible fastest?

**Comment by DAN-256** [comment]
I'll give 35% if it works with first 11 characters search
1ELQy6ZQz7sgB9FDhg9drDSbvP28zMHh8J

**Comment by ishristov** [comment]
> I'll give 35% if it works with first 11 characters search
> 1ELQy6ZQz7sgB9FDhg9drDSbvP28zMHh8J

11 is too much, but I have now idea how fast that ibm will be. In any way it for so many characters it would be faster to not rely on javascript but to have a script in some other language. However, the address you are sharing is almost empty so it's not worth doing anything for it.

**Comment by DAN-256** [comment]
This wallet can help continue graduation or other endeavors 1ELQy6ZQz7sgB9FDhg9drDSbvP28zMHh8J I don't care if you say it's a little bit but I can give at least 35% as a form of thanks if you can help me so that the script can work on ibm's quantum cloud and it works.

**Comment by ishristov** [comment]
> This wallet can help continue graduation or other endeavors 1ELQy6ZQz7sgB9FDhg9drDSbvP28zMHh8J I don't care if you say it's a little bit but I can give at least 35% as a form of thanks if you can help me so that the script can work on ibm's quantum cloud and it works.

11 is too much and I doubt it will work but maybe it's worth trying that ibm cloud, I have no idea what setup do they have. Write me at gallon.uplifting691@anonaddy.me so share more details of what exactly do you have and what have you lost.

**Comment by chenjianquan7** [comment]
> 这个钱包可以帮助继续毕业或其他努力1ELQy6ZQz7sgB9FDhg9drDSbvP28zMHh8J 我不在乎你说它有一点点，但如果你能帮助我，我可以至少给出 35% 作为感谢的形式，以便脚本可以在 ibm 的量子云上运行并且它可以工作。

i can help you , but you must give me 35%,  this is us agreement , please send my email, 963541951@qq.com

**Comment by iamchaay** [comment]
I haven't been trying to do this for a long time, did you see the date when
I posted this message?
